main.py

```diff
@@ -33,10 +73,10 @@ def handle_exception(e):
             "subtitle": f"{e.code}: {e.name}",
             "message": e.description
         }
+        app.logger.warning("handle_exception(): HTTP EXCEPTION")
```

Comment:
```suggestion
        app.logger.warning(f"handle_exception(): {context_dict}")
```
error内容がわかるようにこうするとどうでしょうか？

<img width="1046" alt="Screen Shot 2020-10-12 at 11 37 16" src="https://user-images.githubusercontent.com/40506652/95699417-54eb0380-0c7f-11eb-99b3-b89501642d18.png">

Reply:
そもそも、そのif文の中に入らないとログが残らない方がおかしくない？
ちょっとこっちで書き換えますね。